core/src/com/unciv/ui/screens/worldscreen/unit/actions/UnitActionsFromUniques.kt

```diff
@@ -54,12 +57,19 @@ object UnitActionsFromUniques {
         ) == true }
         val foundAction = {
             if (unit.civ.playerType != PlayerType.AI)
-                UncivGame.Current.settings.addCompletedTutorialTask("Found city")
-            unit.civ.addCity(tile.position, unit)
+                // Now takes on the text of the unique.
+                UncivGame.Current.settings.addCompletedTutorialTask(
+                    unique.text)
```

Comment:
This change breaks the Tutorial "Found a city!" - it will never successfully complete now, as the key this records in GameSettings.tutorialTasksCompleted ("Founds a new city <by consuming this unit>") isn't the one that can clear the event. The literal string was there for a reason. I know this is months old, just recording it for anybody looking. Will fix.